_rules/inline_link_in_paragraph_is_distinguishable_be4d0c.md

```diff
@@ -0,0 +1,240 @@
+---
+id: be4d0c
+name: Inline link in paragraph is distinguishable
+rule_type: atomic
+
+description: |
+  This rule checks that links that are embedded in a paragraph have a way to distinguish them as links compared to the rest of the text
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+acknowledgements:
```

Comment:
Would you mind listing me as a previous author:
https://github.com/act-rules/act-rules.github.io/blob/develop/__deprecated__/_drafts/outdated/SC1-4-1-link-style-in-text.md